res/controllers/Hercules_DJControl_Jogvision.midi.xml

```diff
@@ -0,0 +1,2132 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<MixxxMIDIPreset mixxxVersion="2.2" schemaVersion="1">
+  <info>
+    <name>DJControl Jogvision</name>
+    <author>DJ Phatso for Hercules Technical Support</author>
+    <description>MIDI Mapping for Hercules DJControl Jogvision</description>
+    <forums />
```

Comment:
Please add forum and wiki link

Reply:
https://www.mixxx.org/wiki/doku.php/hercules_dj_control_jogvision

https://www.mixxx.org/forums/viewtopic.php?f=7&t=12580